Microsoft.Toolkit.Uwp.UI.Controls/RadialGauge/RadialGauge.cs

```diff
@@ -45,7 +45,7 @@ public class RadialGauge : Control
         /// Identifies the optional StepSize property.
         /// </summary>
         public static readonly DependencyProperty StepSizeProperty =
-            DependencyProperty.Register(nameof(StepSize), typeof(double), typeof(RadialGauge), new PropertyMetadata(0.0));
```

Comment:
Since we can now handle a zero step-size (like it did originally); I don't think we should change the default.

Reply:
Disagree. A value of zero doesn't even make sense and obviously no one is using zero because it has a bug. We are now handling zero, but default should be 1

Reply:
I agree with @michael-hawker here, we should avoid breaking changes if we can

Reply:
The value of zero didn't work, so it's not a breaking change. Using the value of zero wasn't working. So changing the default from zero to 1 is taking something that wasn't working and is now working

Reply:
@skendrot - StepSize Zero worked fine in the 4.0 release (thus bug #2678), it was introduced as a problem just before 5.0 shipped.  Test with [the commit](https://github.com/windows-toolkit/WindowsCommunityToolkit/commit/c5de98af2ec43e323aea6b50188965b1edf1a0ba#diff-a1347b9e02dd74cb7cf4017627399df1) linked to in the bug.

We should keep the default as zero to not further change behavior and restore the original behavior with the proper fix you've provided.